### ISSUE TITLE
Combining two loss functions

### ISSUE THREAD

**Issue by HarryP23** [KevinMusgrave/pytorch-metric-learning#648]
Hello! I'm trying to combine two loss functions, contrastive loss and arcface loss, in training. The new loss add two result values of each loss. Should I make a new class named CombinedLoss or fix a trainer to add two losses?

**Comment by KevinMusgrave** [comment]
I think `MultipleLosses` would work: https://kevinmusgrave.github.io/pytorch-metric-learning/losses/#multiplelosses

Example:

```python
from pytorch_metric_learning.losses import ContrastiveLoss, ArcFaceLoss, MultipleLosses

loss1 = ContrastiveLoss()
loss2 = ArcFaceLoss(embedding_size=32, num_classes=10)
loss_fn = MultipleLosses([loss1, loss2])
```
